Sources/Operators.swift

```diff
@@ -241,7 +241,7 @@ public func <~~ (key: String, json: JSON) -> [URL]? {
 /**
 Encode custom operator.
 */
-infix operator ~~> { associativity left precedence 150 }
+infix operator ~~> : MultiplicationPrecedence
```

Comment:
One group named GlossPrecedence or DecodingPrecedence?

Or two groups, DecodingPrecedence and EncodingPrecedence?